src/aboutus.html

```diff
@@ -6,22 +6,52 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<meta name="description" content="Update your description here" />
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Jura&display=swap" rel="stylesheet">
 	<title>About Us Page</title>
 </head>
-
-<body>
-	<div class="topnav">
-		<a href="index.html">home</a>
-		<a href="moodselector.html">mood selector</a>
-		<a href="results.html">your music</a>
-		<a class="active" href="aboutus.html">about us</a>
+<body class="aboutus">	
+	<nav>
+		<a href="index.html" class="reftoindex">home</a>
+		<a href="moodselector.html" class="reftomood">mood selector</a>
+		<a href="results.html" class="reftores">your music</a>
+		<a href="aboutus.html" class="reftoact">about us</a>
+	</nav>
+<div class = "projectbox-aboutus">
+	<h4 class = "project-aboutus">THE PROJECT</h4>
+	<p class = "projecttext-aboutus">
+		The goal of this website is to provide users with new music recommendations depending on their current mood.<br><br>
+		Our interactive site takes the search and scrolls out of discovering new music.<br><br>
+		Users will be able to choose between different moods for instantaneous recommendations.<br><br>
+		And as a result, if users like the website's recommendations, they will be able to add them to their Spotify playlists.
+	</p>
+	<div class = "meet-teambox">
+		<h4 class = "meetteamtext">MEET OUR TEAM</h4>
 	</div>
-	<div class="description">
-		<h1>about [insert name]</h1>
-		<p>For this project, we wanted to create a tool for the music lovers. [insert name] takes the search and scroll
-			out of music discovery so you can spend more time listening. Generating a personalized music recommendation is
-			as easy as filling out the survey form.</p>
+		<div class = "member-onebox">
+			<p class = "memberonetext">MELISSA CHAN<br><br>
+				Art Direction, Copy, HTML, CSS, JS<br>
+			</p>
+		</div>
+		<div class = "member-twobox">
+			<p class = "membertwotext">CHLOE QIN<br><br>
+				Art Direction, Copy, HTML, CSS, JS<br>
+			</p>
+		</div>
+		<div class = "member-threebox">
+			<p class = "memberthreetext">MARIA PAVLOVA<br><br>
+				Art & Creative Direction, HTML, CSS<br>
+			</p>
+		</div>
+
+		<div class = "member-fourbox">
+			<p class = "memberfourtext">GARRETT CHOW<br><br>
+				&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Research, HTML, CSS, JS&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br>
```

Comment:
I see there's a lot of `nbsp` here. Is there another way to achieve this style through CSS you could use instead?